container-engine-lib/lib/backend_impls/docker/docker_manager/create_and_start_container_args.go

```diff
@@ -249,3 +255,13 @@ func (builder *CreateAndStartContainerArgsBuilder) WithContainerInitEnabled(cont
 	builder.containerInitEnabled = containerInitEnabled
 	return builder
 }
+
+func (builder *CreateAndStartContainerArgsBuilder) WithPullingLatestImageAlways() *CreateAndStartContainerArgsBuilder {
```

Comment:
I like that you separated the two functions in two as opposed to having the `image download` mode passed in! 

also very nit smth I care about having consistent naming of this behavior across the repo - I think `Pulling` makes sense here but now we're using `download` in CLI, `Fetch` in `container-engine-lib` and `Pulling` here to describe the same behavior. I wonder if there's a way to simplify this 

no ask here jus wanted to point out

Reply:
maybe I'd consider changing `Pulling` to `Fetch` so we can keep it to `download` and `fetch` but up to you